transports/quic/src/lib.rs

```diff
@@ -59,11 +59,15 @@
 
 mod connection;
 mod endpoint;
+mod hole_punching;
 mod provider;
 mod transport;
 
+use std::net::SocketAddr;
+
 pub use connection::{Connecting, Connection, Substream};
 pub use endpoint::Config;
+use libp2p_identity::PeerId;
```

Comment:
```suggestion
use std::net::SocketAddr;
use libp2p_identity::PeerId;

pub use connection::{Connecting, Connection, Substream};
pub use endpoint::Config;
```

Let's not mix private and public imports.